framework/web/Request.php

```diff
@@ -89,10 +89,6 @@ class Request extends \yii\base\Request
      * The name of the HTTP header for sending CSRF token.
      */
     const CSRF_HEADER = 'X-CSRF-Token';
-    /**
-     * The length of the CSRF token mask.
-     */
-    const CSRF_MASK_LENGTH = 8;
```

Comment:
Anyone depending on it is most likely doing something related to masking / unmasking the token, so their code should then break. It is a (minor) BC break though. Will update OP to reflact that.